test/facets/Token/behaviors/enableSupportedChains.t.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.19;
+
+import { TokenBridge } from "../TokenBridge.t.sol";
+import { ArbForkTest } from "../../../ArbForkTest.t.sol";
+import { ITokenBridge } from "../../../../contracts/facets/Token/ITokenBridge.sol";
+
+/// @title EnableSupportedChains
+/// @notice This contract tests the functionalities of the enableSupportedChains function
+contract EnableSupportedChains is ArbForkTest, TokenBridge {
```

Comment:
Please edit to match styling comments above for contract + test function names.

Reply:
fixed here https://github.com/giovannifranchi/insrt-v2-contracts/commit/4d1fccbd8f21a673e658c6e216ac59573bb3e306